### PR TITLE
Auto bump AssetLastHostname

### PR DESCRIPTION
**Pull request by ghost**
Auto bump counter in `AssetLastHostname` to value from `current_counter_without_model` when `network_env.use_hostname_counter` changed to `True`.

### CODE REVIEW

**Comment by xliiv** [comment]
what if operation is reversed?

File: src/ralph/networks/models/networks.py [hunk 1]
```diff
@@ -719,3 +739,20 @@ def rebuild_handler(sender, **kwargs):
...
+
+
+@receiver(pre_save, sender=NetworkEnvironment)
+def update_counter(sender, instance, **kwargs):
+    if not instance.id:
+        return
+
+    orig = NetworkEnvironment.objects.get(id=instance.id)
+    if instance.use_hostname_counter != orig.use_hostname_counter:
+        counter = AssetLastHostname(
```
Comment (mkurek) [inline src/ralph/networks/models/networks.py]: try update_or_create

**Comment by ar4s** [comment]
:ok:

File: src/ralph/networks/tests/test_models.py [hunk 2]
```diff
@@ -361,6 +361,28 @@ def test_next_hostname_without_counter(self):
...
+    def test_use_hostname_counter_updates_last_hostname_counter(self):
+        prefix = 'test.'
+        postfix = '.ralph.pl'
+        network_env = NetworkEnvironmentFactory(
+            hostname_template_prefix=prefix,
+            hostname_template_postfix=postfix,
+            hostname_template_counter_length=6,
+            use_hostname_counter=False,
+        )
+        current = 1
```
Comment (mkurek) [inline src/ralph/networks/tests/test_models.py]: could you set current to some value greater than 1 (ex. 10)? :)

**Comment by mkurek** [comment]
:ok:

**Pull request merged**